### ISSUE TITLE
How to dump to sqlite file?

### ISSUE THREAD

**Issue by ununian** [kikko-land/kikko#64]
and import sqlite file

**Comment by ununian** [comment]
and import sqlite file

**Comment by quolpr** [comment]
For which platform? Web, desktop, mobile?

**Comment by astahmer** [comment]
I was also wondering how to do that for the web platform ! I figured it might already be possible using something like https://sql.js.org/#/?id=write-a-database-to-the-disk ? but idk if we can use that `export` function with `@kikko-land/sql.js`

**Comment by ununian** [comment]
web platform。Thanks for your reply

**Comment by quolpr** [comment]
@astahmer yeah, I think I will expose API for dumping and loading SQLite db

**Comment by ununian** [comment]
@quolpr  
Can you expose api for reading and writing files?
I want to write files in indexeddb before `initDbClient`

**Comment by quolpr** [comment]
@ununian I will add support once I finish this PR https://github.com/kikko-land/kikko/pull/61 , that adds massive changes to the code. 

You will be able to call before `initDbClient`, the API will look something like this:

```
const backend = absurdWebBackend({
  wasmUrl: sqlWasmUrl,
});

const config: IInitDbClientConfig = {
  dbName: "db-name",
  dbBackend: backend,
  plugins: [
    migrationsPlugin({ migrations: [createNotesTableMigration] }),
    reactiveQueriesPlugin(),
  ],
};

const data = backend.export() // exporting DB Uint8Array
backend.load(data) // loading DB
```
